src/dcqc/suites/suite_abc.py

```diff
@@ -70,10 +70,10 @@ def list_test_classes(cls) -> tuple[Type[TestABC], ...]:
         for cls in reversed(superclasses):  # Start from the base class
             if hasattr(cls, "add_tests"):
                 add_tests = set(cls.add_tests)  # type: ignore
-                all_tests.union(add_tests)
+                all_tests |= add_tests
```

Comment:
TIL: these short cuts

Reply:
I didn't like having to reassign the output of `union()` to the variable. 

```python
all_tests = all_tests.union(add_tests)
```

That said, I found the `*update()` set functions, so I'm using that instead. 

```python
if hasattr(cls, "add_tests"):
    add_tests = set(cls.add_tests)  # type: ignore
    all_tests.update(add_tests)
if hasattr(cls, "del_tests"):
    del_tests = set(cls.del_tests)  # type: ignore
    all_tests.difference_update(del_tests)
```

Reply:
I'll make this change in the following PR.

Reply:
TIL: `difference_update`